app/src/main/res/values/dimens.xml

```diff
@@ -43,4 +43,7 @@
     <!-- Default screen margins, per the Android Design guidelines. -->
     <dimen name="activity_horizontal_margin">16dp</dimen>
     <dimen name="activity_vertical_margin">16dp</dimen>
+
+    <item name="setup_buttons_top_guideline" type="dimen">0.868</item>
+    <item name="setup_buttons_bottom_guideline" type="dimen">0.952</item>
```

Comment:
Just from curiosity, why did you put only this two values in dimens? 🤔

Reply:
Because they were used everywhere with the setup buttons. During the development those values were changed for some of the buttons and the they weren't properly aligned when going from one view to another. So i introduced unified values in dimens in order to keep them aligned while transitioning from view to view.